src/background/messaging/rpc-methods/sign-psbt.ts

```diff
@@ -60,7 +61,6 @@ export async function rpcSignPsbt(message: SignPsbtRequest, port: chrome.runtime
   const requestParams: RequestParams = [
     ['requestId', message.id],
     ['hex', message.params.hex],
-    ['publicKey', message.params.publicKey],
```

Comment:
Making note here, on L64 below, I don't think `network` is used anywhere in the implementation?

Reply:
@kyranjamie what do you think here, `network` can be removed, correct?

Reply:
Network is used in this implementation

https://github.com/hirosystems/wallet/blob/7b1b1fde1f9afed7ff7e6d56f7aa35f5c07c090e/src/app/store/networks/networks.ts#L17

Reply:
Added network back to psbt signing, and put up this PR in btckit: https://github.com/btckit-org/btckit/pull/5

Pretty sure we should add an optional network param to the others here?